mbed_targets/_internal/target_database.py

```diff
@@ -1,15 +1,34 @@
 """Internal helper to retrieve target information from the online database."""
 
-from http import HTTPStatus
+import os
 
+from http import HTTPStatus
 from json.decoder import JSONDecodeError
 from typing import List
 
+import dotenv
 import requests
 
 
+# Search for the .env file containing the MBED_API_AUTH_TOKEN environment variable.
+# We want this to execute at import time.
+dotenv.load_dotenv(dotenv.find_dotenv(usecwd=True), override=True)
+
+
+def _set_target_api_url() -> str:
```

Comment:
I'm trying to think of a use case where a user would want to change this, but I haven't so far. If we can't think of a use case I don't think we should make it configurable.

Reply:
Surely this should be '_get, not '_set'? Also looks a bit over-engineered to me; YAGNI?

Reply:
Also, why not os.getenv("MBED_TARGET_API_URL", "https://os.mbed.com/api/v4/targets")? Forgive my python noob question here.